matchstart.fbs

```diff
@@ -75,9 +75,8 @@ table PlayerConfiguration {
   location:string (required);
```

Comment:
What do you think of renaming `PlayerConfiguration::location` and `ScriptConfiguration::location` to `root_dir`? In other contexts, location is a vector.

Reply:
Sure